src/components/20-molecules/top-content-bar/index.js

```diff
@@ -12,7 +14,7 @@ import {
 import { applyDefaults } from '../../../utils/with-react';
 import styles from './index.scss';
 
-class AXATopContentBar extends LitElement {
+class AXATopContentBar extends InlineStyles {
```

Comment:
I would like to see a brief comment here in the code for why InlineStyles.

Reply:
Please re-check if this is good enough now or give an example how you would rename it.

Reply:
I did, see above